api/apis/process_handler.go

```diff
@@ -142,16 +132,8 @@ func (h *ProcessHandler) processScaleHandler(authInfo authorization.Info, w http
 
 	processRecord, err := h.scaleProcess(ctx, authInfo, processGUID, payload.ToRecord())
 	if err != nil {
-		switch err.(type) {
-		case repositories.NotFoundError:
-			h.logger.Info("Process not found", "processGUID", processGUID)
-			writeNotFoundErrorResponse(w, "Process")
-			return
-		default:
-			h.logger.Error(err, "Failed due to error from Kubernetes", "processGUID", processGUID)
-			writeUnknownErrorResponse(w)
-			return
-		}
+		h.logError(w, processGUID, err)
```

Comment:
ScaleProcess is going to be picked up in #623. Although this is obviously the correct code, it's not yet covered by the process handler tests, so I think it should be left as is until that story is played.